spec/compiler/semantic/def_spec.cr

```diff
@@ -272,7 +272,7 @@ describe "Semantic: def" do
 
       foo
       ),
-      "method must return Int32 but it is returning Char"
+      "method top-level foo must return Int32 but it is returning Char"
```

Comment:
Is it worth it to phrase this as "top-level method foo" instead?

Reply:
I was rather thinking of removing `top-level` from `Def.short_reference` and just use a dot instead: `method .foo must return ...`.
But that's a different change because it also affects other messages, so I refrained from changing that here directly.